File: coding/Codeforces/round_815/task_A.py
# https://codeforces.com/contest/1720/problem/A

def solving(a, b, c, d):
    if (a == 0 and c != 0) or (a != 0 and c == 0):
        return 1
    elif (a == 0 and c == 0) or (a / c == b / d):
        return 0
    elif (
            ((a / c) / (b / d) == int((a / c) / (b / d))) or
            ((c / a) / (d / b) == int((c / a) / (d / b)))):
        return 1
    else:
        return 2

def main():
    t = int(input())
    answer = []
    for _ in range(t):
        a, b, c, d = map(int, input().split())
        answer.append(solving(a, b, c, d))
    answer = "\n".join(map(str, answer))
    print(answer)

if __name__ == "__main__":
    main()

File: coding/Codeforces/round_930/task_A.cpp
// https://codeforces.com/contest/1933/problem/A
// Created by ruslan515 on 27.02.2024
//

#include<bits/stdc++.h>

using namespace std;

void solving() {
    int n;
    cin >> n;
    int answer = 0;
    vector<int> a(n);
    for (int i = 0; i < n; ++i) {
        cin >> a[i];
        answer += abs(a[i]);
    }

    cout << answer << "\n";
    return;
}

int main() {
    ios::sync_with_stdio(false);
    ios_base::sync_with_stdio(false);
    cin.tie(0);

    int t;
    cin >> t;
    for (int i = 0; i < t; ++i) {
        solving();
    }

    return 0;
}

